### ISSUE TITLE
Break out sales statistics by item

### ISSUE THREAD

**Issue by harrislapiroff** [littleweaver/django-brambling#818]
It would be really great for organizers to see how much total money they make from each item in their shop. (E.g., if an event sells both event passes and t-shirts, it would be useful on the dashboard to see that they made $X from passes and $Y from t-shirts.)

**Comment by harrislapiroff** [comment]
(Feature requested by an organizer)

**Comment by melinath** [comment]
We do already have the summary page, but we could make it a more official table (i.e. name, cost per unit, units sold, $$$)

Current sales summary:

![screen shot 2016-08-08 at 9 13 24 am](https://cloud.githubusercontent.com/assets/299979/17482671/7f99ced0-5d48-11e6-9b8c-5b656939e5c0.png)

**Comment by harrislapiroff** [comment]
Yeah, I pointed the organizer to that summary, but the specific request was to see $ totals.

**Comment by harrislapiroff** [comment]
nice/fun
